boa/boa-id-v1.jsonld

```diff
@@ -0,0 +1,35 @@
+{
+  "@context":
+  {
+    "@version": 1.1,
+    "bid": "https://sphereon-opensource.github.io/factom-vc-contexts/boa/boa-id-v1#",
+    "xsd": "http://www.w3.org/2001/XMLSchema#",
+    "schema": "http://schema.org/",
+
+    "BoaIdCredential" : "bid:BoaIdCredential",
+
+    "issueDate": {
+      "@id": "bid:issueDate",
+      "@type": "xsd:dateTime"
+    },
+    "validityDuration": {
+      "@id": "bid:validityDuration",
+      "@type": "schema:duration"
+    },
+    "name": {
+      "@id": "bid:name",
+      "@type": "schema:name"
+    },
+    "boaNumber": "bid:boaNumber",
+    "idPhoto": {
+      "@id": "bid:idPhoto",
+      "@type": "schema:image"
+    },
+    "domain": "bid:domain",
+    "employer": "bid:employer",
+    "locality": {
+      "@id": "bid:locality",
+      "@type": "schema:administrativeArea"
```

Comment:
Probably a text value for now would be enough. Something to discuss I guess

Reply:
Yeah, I wanted to connect these values as much as possible to objects defined in `schema` and this was the closest. For now I can revert it to just a text value.

Reply:
Why not make BOA a Person or even Employee then? ;)

Reply:
Yes, the way I've seen this done is to add `"Person":"schema:person"` to the context and then in the credential itself, you would specify `"@type":"Person"` in the credential subject. I've added this here and tested it in JSON-LD playground, but I'm not sure if there is a better way to enforce on a context level that a BOA is a person.

Reply:
Yeah that is the common pattern